dev-tools/mage/target/update/update.go

```diff
@@ -4,9 +4,28 @@
 
 package update
 
-import "github.com/magefile/mage/sh"
+import (
+	"fmt"
 
-// Update updates the generated files (aka make update).
-func Update() error {
```

Comment:
This target wasn't used, and the make target it calls doesn't exist, so I deleted it.